### PR TITLE
Update zap stanza in android-studio

### PR DESCRIPTION
**Pull request by nrlquaker**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/android-studio.rb [hunk 1]
```diff
@@ -15,9 +15,14 @@
                 '~/Library/Android/sdk',
```
Comment (commitay) [inline Casks/android-studio.rb]: This line can be moved to `trash`

File: Casks/android-studio.rb [hunk 2]
```diff
@@ -15,9 +15,14 @@
...
                 "~/Library/Application Support/AndroidStudio#{version.major_minor}",
```
Comment (commitay) [inline Casks/android-studio.rb]: Remove this line.

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/android-studio.rb (hunk 1 above)
Comment (nrlquaker) [reply Casks/android-studio.rb]: Done.

File: Casks/android-studio.rb (hunk 2 above)
Comment (nrlquaker) [reply Casks/android-studio.rb]: Done.

**Review by commitay — APPROVED** [review]

**Pull request merged**